portability-core/src/test/java/org/dataportabilityproject/serviceProviders/flickr/FlickrPhotoServiceTest.java

```diff
@@ -0,0 +1,126 @@
+
+import com.flickr4java.flickr.Flickr;
+import com.flickr4java.flickr.FlickrException;
+import com.flickr4java.flickr.auth.Auth;
+import com.flickr4java.flickr.auth.Permission;
+import com.flickr4java.flickr.people.User;
+import com.flickr4java.flickr.photos.Photo;
+import com.flickr4java.flickr.photos.PhotosInterface;
+import com.flickr4java.flickr.photos.Size;
+import com.flickr4java.flickr.photosets.Photoset;
+import com.flickr4java.flickr.photosets.Photosets;
+import com.flickr4java.flickr.photosets.PhotosetsInterface;
+import com.flickr4java.flickr.uploader.Uploader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import org.dataportabilityproject.cloud.interfaces.JobDataCache;
+import org.dataportabilityproject.dataModels.ExportInformation;
+import org.dataportabilityproject.dataModels.photos.PhotoAlbum;
+import org.dataportabilityproject.dataModels.photos.PhotoModel;
+import org.dataportabilityproject.dataModels.photos.PhotosModelWrapper;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class FlickrPhotoServiceTest {
+
+  static final String PHOTO_TITLE = "Title";
+  static final String FETCHABLE_URL = "fetchable_url";
+  static final String DESCRIPTION = "Description";
+  static final String MEDIA_TYPE = "jpeg";
+
+  static final String ALBUM_ID = "Album ID";
+  static final String ALBUM_NAME = "Album name";
+  static final String ALBUM_DESCRIPTION = "Album description";
+
+  static final PhotoModel PHOTO_MODEL = new PhotoModel(PHOTO_TITLE, FETCHABLE_URL, DESCRIPTION,
+      MEDIA_TYPE, ALBUM_ID);
+  static final PhotoAlbum PHOTO_ALBUM = new PhotoAlbum(ALBUM_ID, ALBUM_NAME, ALBUM_DESCRIPTION);
+
+  Flickr flickr = Mockito.mock(Flickr.class);
+  PhotosetsInterface photosetsInterface = Mockito.mock(PhotosetsInterface.class);
+  PhotosInterface photosInterface = Mockito.mock(PhotosInterface.class);
+  Uploader uploader = Mockito.mock(Uploader.class);
+  JobDataCache jobDataCache = Mockito.mock(JobDataCache.class);
+  User user = Mockito.mock(User.class);
+  Auth auth = new Auth(Permission.WRITE, user);
+  FlickrPhotoService photoService = new FlickrPhotoService(flickr, photosetsInterface,
+      photosInterface, uploader, jobDataCache, auth);
+
+  @Test
+  public void getPage() {
+    // No pagination information, so we must be on page 1
+    assertThat(FlickrPhotoService.getPage(Optional.empty())).isEqualTo(1);
+
+    // Correctly get current page
+    int page = 6;
+    FlickrPaginationInformation paginationInformation = new FlickrPaginationInformation(page);
+    assertThat(FlickrPhotoService.getPage(Optional.of(paginationInformation))).isEqualTo(page);
+  }
+
+  @Test
+  public void getMimeType() {
+    assertThat(FlickrPhotoService.toMimeType("jpeg")).isEqualTo("image/jpeg");
+    assertThrows(IllegalArgumentException.class, () -> FlickrPhotoService.toMimeType("png"));
+  }
+
+  @Test
+  public void toCommonPhoto() {
+    Photo photo = new Photo();
```

Comment:
Feel free to move this test photo object creation logic into a helper method if you think we'll use it for other tests in this class

Reply:
Done.